### ISSUE TITLE
Import reverse object relation

### ISSUE THREAD

**Issue by ascheider** [BlackbitDigitalCommerce/pimcore-data-director#166]
Is the import on the reverse object relation side currently possible?
If yes, how?
There is no template selection in the field configuration.
i tried to load the objects and return the array of them, but i get the invalid mapping error.

[INFO] Value for field productAssignment: [
    "/Product Catalogue/France Products/3M™ Handroller für Sicken"
]
[WARNING] Unable to set value for field "productAssignment", Invalid mapping for field productAssignment: No valid target found

**Comment by BlackbitDevs** [comment]
No, this is currently not possible. A long time ago I had implemented the setter methods for reverse relations in https://github.com/pimcore/pimcore/pull/4262 but it got rejected (at least the getter methods got merged in the follow-up PR https://github.com/pimcore/pimcore/pull/10242).
Of course we can implement support for importing into reverse relatins. Just write a mail to help@blackbit.com
Alternatively you could call a dependent dataport and set the relations on the owner side.
